apps/condo/domains/user/schema/User.js

```diff
@@ -190,6 +194,7 @@ const User = new GQLListSchema('User', {
                             })
                         }
                         if (existedUsers && existedUsers.length > 0) {
+                            // TODO(DOMA-9749): migrate from addFieldValidationError to GQLError
```

Comment:
Why not do it in this pr?

Reply:
I don't want the task to become twice as big)